test/pylib/scylla_cluster.py

```diff
@@ -993,7 +993,8 @@ async def _after_test(self, _request) -> aiohttp.web.Response:
         finally:
             self.current_test_case_full_name = ''
         self.is_after_test_ok = True
-        return aiohttp.web.Response(text="True")
+        cluster_str = str(self.cluster)
+        return aiohttp.web.Response(text=cluster_str)
```

Comment:
I'm not sure I understand why printing "the cluster" (I assume this looks like a list of node IPs? Or something?) is a useful thing to do after every test, but seeing that you and the other reviewers are actually using this code and looking and these messages and like it, I'll accept.

Reply:
Yes, it shows the list of nodes (their "test IDs" - which the directories are named after, IPs, and Host IDs).

I heard complaints that it's hard to find the cluster used by the test. You could find the cluster by looking near the beginning of the log, but it was a bit annoying to scroll and it didn't show the latest list of nodes (the cluster might've been modified by the test, some nodes added, some removed).